# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kde/kdebase/konsole-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN ./konsole/other.orig/linux.desktop ./konsole/other/linux.desktop
--- ./konsole/other.orig/linux.desktop	2004-03-12 16:20:32.000000000 -0500
+++ ./konsole/other/linux.desktop	2004-03-12 16:22:15.000000000 -0500
@@ -120,7 +120,7 @@
 Comment[zh_CN]=新建 Linux 控制台
 Comment[zh_TW]=新增 Linux 主控台
 Comment[zu]=Ikhonsoli Entsha ye-Linux
-Exec=bash
+Exec=bash --login
 Schema=Linux.schema
 #VGA
 Font=6
diff -urN ./konsole/other.orig/shell.desktop ./konsole/other/shell.desktop
--- ./konsole/other.orig/shell.desktop	2004-03-12 16:20:32.000000000 -0500
+++ ./konsole/other/shell.desktop	2004-03-12 16:22:15.000000000 -0500
@@ -108,5 +108,5 @@
 Comment[zh_CN]=新建 Shell
 Comment[zh_TW]=新增 Shell
 Comment[zu]=Igobolodo Elisha
-Exec=
+Exec=bash --login
 Cwd=
diff -urN ./konsole/other.orig/su.desktop ./konsole/other/su.desktop
--- ./konsole/other.orig/su.desktop	2004-03-12 16:20:32.000000000 -0500
+++ ./konsole/other/su.desktop	2004-03-12 16:22:15.000000000 -0500
@@ -121,7 +121,7 @@
 Comment[zh_CN]=新建 Root 控制台
 Comment[zh_TW]=新增 Root 主控台
 Comment[zu]=Ikhonsoli Yempande Entsha
-Exec=su
+Exec=su -
 Schema=BlackOnLightYellow.schema
 #Schema=Linux.schema
 #VGA
